src/lib/common/sem.cpp

```diff
@@ -45,6 +46,15 @@ static SemRequestType  reqPolicy;
 
 /* ****************************************************************************
 *
+* Time measuring variables - 
+*/
+static struct timespec accReqSemTime   = { 0, 0 };
+static struct timespec accMongoSemTime = { 0, 0 };
+
```

Comment:
I think we should also do the same for the transaction semaphore (I forget about it when I write #911)